httpx/middleware/newrelic_tracer.go

```diff
@@ -0,0 +1,59 @@
+package middleware
+
+import (
+	"fmt"
+	"github.com/remind101/nra"
+	"github.com/remind101/pkg/httpx"
+	"golang.org/x/net/context"
+	"net/http"
+)
+
+type NewRelicTracer struct {
```

Comment:
In the docs for this, might wanna mention that this needs to be added downstream from the httpx.Router; so wrapping an httpx.Router won't work, you need to add it when adding a handler: `r.Handle(NewRelicTracing(h, ...))`.